spec/default_spec.rb

```diff
@@ -9,6 +9,33 @@
       it 'converges successfully' do
         expect { chef_run }.to_not raise_error
       end
+      it do
+        expect(chef_run.node['varnish']['version']).to eq(3)
+      end
+      it do
+        expect(chef_run.node['varnish']['log_daemon']).to eq(false)
+      end
+      it do
+        expect(chef_run).to include_recipe('varnish')
+      end
+      it do
+        expect(chef_run).to create_template('/etc/sysconfig/varnishncsa').with(
+          source: 'varnishncsa.erb',
+          variables: { directives: chef_run.node['osl-varnish']['ncsa_directives'] }
```

Comment:
Can you please add another test using ``render_file`` to ensure this gets generated properly? You might also make a new context to change the attributes to ensure it's doing what you expect.